src/data/navigation.ts

```diff
@@ -55,6 +55,7 @@ export const navigation: NavigationItem[] = [
 						subsections: [
 							{ title: "Chat", href: "/cody/capabilities/chat", },
 							{ title: "Autocomplete", href: "/cody/capabilities/autocomplete", },
+							{ title: "Auto-edits", href: "/cody/capabilities/auto-edits", },
```

Comment:
After some pondering, we [agreed](https://www.notion.so/sourcegraph/FAQ-6-0-Release-Jan-29-2024-159a8e112658804ca222f7073427ce10) on the `auto-edit` feature name. 

```suggestion
							{ title: "Auto-edit", href: "/cody/capabilities/auto-edit", },
```